doc/en/weechat_faq.en.asciidoc

```diff
@@ -63,7 +63,8 @@ brew info weechat
 You can install WeeChat with this command:
 
 ----
-brew install weechat --with-python --with-perl
+brew install aspell
+brew install weechat --with-aspell --with-curl --with-guile --with-lua --with-perl --with-python --with-ruby
```

Comment:
Actually the homebrew formula doesn't mention tcl either. https://github.com/Homebrew/homebrew/blob/master/Library/Formula/weechat.rb